.github/workflows/e2e-tf-deployment.yaml

```diff
@@ -0,0 +1,80 @@
+name: Helm Charts E2E Testing
```

Comment:
Should we call this: 

`Bamboo Helm Charts E2E Testing`

Reply:
This is supposed to be more generic as we are going to add more products. So I didn't use `Bamboo` to make it product specific.